src/scripts/audit/tool.sh

```diff
@@ -0,0 +1,305 @@
+#!/bin/bash
+# Copyright 2020, Oracle Corporation and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+
+#set -x
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/utility.sh
+
+function usage {
+  echo "usage: ${script} [-i] [-s] [-w] [-h]"
+  echo "  -i File containing list of images to scan in [Repository]:[Tag]@[Hash] format."
```

Comment:
Will the list be one-per-line or comma-separated, etc.?

Reply:
It'll be one-per-line, I'll mention this in the usage to clarify this.